Smt/Reconstruct/Real.lean

```diff
@@ -265,6 +265,10 @@ def reconstructMulAbsComparison (pf : cvc5.Proof) : ReconstructM (Option Expr) :
       let h : Q(|$l| = |$r|) ← reconstructProof p
       return (.EQUAL, lsl, rsr, q(Real.mul_abs₁ $hs $h))
     else if ks == .GT && k == .AND then
+      let l : Q(Real) ← reconstructTerm p.getResult[0]![0]![0]!
+      let r : Q(Real) ← reconstructTerm p.getResult[0]![1]![0]!
+      let lsl := q($ls * $l)
+      let rsr := q($rs * $r)
```

Comment:
I think it's better to just have the lambda function start with the conditions. So, move lines 258-261 after 263 and 275.